src/App.tsx

```diff
@@ -1,173 +1,65 @@
-
-                  <td>
-                    <a href="#/people/emile-haverbeke-1877">
-                      Emile Haverbeke
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person" className="has-background-warning">
-                  <td>
-                    <a href="#/people/jan-frans-van-brussel-1761">
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
+import { LoadingError } from './LoadingError';
+import { MainNav } from './MainNav';
+import { PageNotFound } from './PageNotFound';
+import { PeoplePage } from './PeoplePage';
+import { Person } from './types';
+
+export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [usersArr, setUsersArr] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const getUsers = async () => {
+      try {
+        setIsLoading(true);
+
+        const response = await fetch(
+          'https://mate-academy.github.io/react_people-table/api/people.json',
+        );
+
+        const responseData = await response.json();
+
+        setUsersArr(responseData);
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.log('error');
```

Comment:
In production code, avoid console.log statements. To handle errors, consider implementing a user-friendly error message.